### PR TITLE
Add API provider page

### LINKED ISSUE #29: Add new page: API providers
### Description
Summarize available API providers.

### Category
Lisk tools

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?

This PR resolves #29 

### How was it tested?

https://liskhq.github.io/lisk-documentation/

### CODE REVIEW

File: docs/lisk-tools/api-providers.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    API,
+    Web3 infrastructure,
+  ]
+---
+
+# API Providers
+
+Lisk nodes expose an RPC API that allows other parties to interact with the blockchain by invoking requests.
+
+If you're just getting started and need an RPC URL, you can use our [free endpoints](#Lisk-RPC).
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/api-providers.md]:
```suggestion
If you're just getting started and need an RPC URL, you can use our [free endpoints](#lisk-rpc).
```

File: docs/lisk-tools/api-providers.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+If you're looking to strenghten your app and avoid rate-limiting for your users, please check out our available RPC node providers like [dRPC](#dRPC).
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/api-providers.md]:
```suggestion
If you're looking to strengthen your app and avoid rate-limiting for your users, please check out our available RPC node providers like [dRPC](#drpc).
```

File: docs/lisk-tools/api-providers.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+|               | Lisk Sepolia Testnet              | Lisk Mainnet              |
+| :------       | :------                           | :-----------------------  |
+|**HTTP RPC**   | https://rpc.sepolia-api.lisk.com  | https://rpc.api.lisk.com  |
+|**WS RPC**     | `wss://ws.sepolia-api.lisk.com`   | `wss://ws.api.lisk.com`   |
+
+
+## dRPC
+
+[dRPC](https://drpc.org/) is a decentralized Web3 infrastructure provider with focus on resilience and latency.
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/api-providers.md]:
```suggestion
[dRPC](https://drpc.org/) is a decentralized Web3 infrastructure provider with a focus on resilience and latency.
```

File: docs/lisk-tools/api-providers.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+dRPC offers access to a distributed network of public nodes for Lisk.
+They provide a free tier that allows for an unlimited amount of requests over public nodes, or a paid tier which provides access to all providers, as well as other additional features.
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/api-providers.md]:
```suggestion
They provide a free tier that allows for an unlimited amount of requests over public nodes, or a paid tier that provides access to all providers, as well as other additional features.
```

File: docs/lisk-tools/api-providers.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    archival nodes,
+    RPC,
+    RPC node,
+    RPC URL,
+    RPC endpoints,
+    blockchain services,
+    blockchain infrastructure,
+    developer tools,
+    API,
+    Web3 infrastructure,
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/api-providers.md:21]:
```suggestion
    Web3 infrastructure,
    dRPC,
    Sepolia,
```

File: docs/lisk-tools/api-providers.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+|**WS RPC**     | `wss://ws.sepolia-api.lisk.com`   | `wss://ws.api.lisk.com`   |
+
+
+## dRPC
+
+[dRPC](https://drpc.org/) is a decentralized Web3 infrastructure provider with focus on resilience and latency.
+dRPC offers access to a distributed network of public nodes for Lisk.
+They provide a free tier that allows for an unlimited amount of requests over public nodes, or a paid tier which provides access to all providers, as well as other additional features.
+
+Check the available endpoints for Lisk directly under [https://drpc.org/public-endpoints/lisk](https://drpc.org/public-endpoints/lisk)
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/api-providers.md]:
```suggestion
Check the available endpoints for Lisk directly under [https://drpc.org/public-endpoints/lisk](https://drpc.org/public-endpoints/lisk).
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Please consider the following.

File: docs/lisk-tools/api-providers.md (hunk 3 above)
Comment (Tschakki) [reply docs/lisk-tools/api-providers.md]: I dont see why `a` is needed here. Their original docs also dont have it: https://docs.drpc.org/

Comment (TalhaMaliktz) [reply docs/lisk-tools/api-providers.md]: Because the noun `focus` is missing an article which in this case is an `a`.

Comment (Tschakki) [reply docs/lisk-tools/api-providers.md]: I dont think it is needed?

Comment (TalhaMaliktz) [reply docs/lisk-tools/api-providers.md]: It is :)

**Review by TalhaMaliktz — APPROVED** [review]
Thank you! LGTM 👍🏼

**Pull request merged**